apps/pie-docs/src/assets/styles/components/_anchorHeading.scss

```diff
@@ -0,0 +1,47 @@
+@use 'sass:color';
+@use '../fozzie-config' as f;
+
+$anchor-size: f.spacing(e);
+$anchor-margin: f.spacing(b);
+
+.c-anchor-icon {
```

Comment:
Small one – the naming is a bit conflicted at then moment. The filename is for `anchorHeading` and the component is `c-anchor-icon`.

I'd change them to match – I think just `anchor` or `anchorIcon` makes sense tbh, so you could just go for `_anchorIcon.scss` and `c-anchorIcon`

Reply:
Hey @ashleynolan, agreed, and i tried this. But as I have now added the span around the heading text I'm not sure if the naming still works?

I've gone for 
file name: `anchor.scss`
icon class: `c-anchor-icon`
heading text `c-anchor-text`

Not confident I've nailed it there so very happy to change it up again. What do you think?

Reply:
Ignore this 😄